# Published by mobilenet_detection

Header header

float32 score   # confidence score of detection, 1.0 is max (0.5 is usually positive detection threshold)
float32 dist    # distance to object (meters)
float32 left_edge  # x-coordinate of left edge of bounding box (in pixels)
float32 right_edge # x-coordinate of right edge of bounding box (in pixels)